Source/Executables/Validation/CMakeLists.txt

```diff
@@ -82,6 +82,7 @@ if (Katydid_ENABLE_TESTING)
         TestSpectrogramStriperSwaps
         TestSpectrumDiscriminator
         TestWindowFunction
+        TestSequentialTrackFinder
```

Comment:
These are listed alphabetically.  Please move it up a few lines to the appropriate place.